image_data/office/ReadMe.md

```diff
@@ -0,0 +1,9 @@
+# Office dataset
+
+Office-31 + Caltech-10 dataset [[Original data source](http://www.stat.ucla.edu/~jxie/iFRAME/code/imageClassification.rar)]. Folder names (i.e. class names) of Caltech-10 from the original source have been renamed to be consistent with the folder names of Office-31.
+
+## Reference
+
+[1] Saenko, K., Kulis, B., Fritz, M. and Darrell, T., 2010, September. Adapting visual category models to new domains. In *European conference on computer vision* (pp. 213-226). Springer, Berlin, Heidelberg.
+
+[2] Griffin, Gregory and Holub, Alex and Perona, Pietro, 2007. Caltech-256 Object Category Dataset. California Institute of Technology. (Unpublished). https://resolver.caltech.edu/CaltechAUTHORS:CNS-TR-2007-001.
```

Comment:
Are you sure this is the correct ref? http://www.vision.caltech.edu/Image_Datasets/Caltech101/

Reply:
See also https://www.cc.gatech.edu/~judy/domainadapt/

Reply:
Please see the first paragraph in 4.1 and reference 15 of [Gong et al. (2012)](https://www.cs.utexas.edu/~grauman/papers/subspace-cvpr2012.pdf), which is also cited in Judy's link above.